### DISCUSSION TITLE
Simulator lagging when Autoortho is downloading a new area[Question]

### DISCUSSION THREAD

**Discussion by PilotFidi** [kubilus1/autoortho#390] [Q&A]
### Please check existing docs and issues first.

- [X] I have checked for existing answers
- [X] This is not a bug or other "Issue"
- [X] I have read the FAQ and documentation

### System Details

- OS: [Windows 10]
- XPlane Version: [XP12 beta 6]
- AutoOrtho Version [e.g. 0.6.1]

### question

So now I got a question for you all, I have a 1 TB SSD, good wifi, good frames whenever im not loading a new tile, but when loading a new tile my whole simulator goes to around 10-15 fps, causing vatsim etc to disconnect which is super frustrating. Does anyone have an idea how to avoid this / fix this? 
Thank you for the help! (:

**Comment by NNowakowski** [comment]
You don't. There's no way to avoid this sadly from what I know.

**Reply by PilotFidi** [reply]
How can it be that other people dont experience this issue though?

**Comment by karlrado** [comment]
I had the same problem when running with VATSIM.  When a new tile got loaded, the CPU usage of AO would increase and XP frame rate would drop to just below 20 fps, causing VATSIM to disconnect. I normally achieve 30 easily and have it capped at 30.

I changed the AO configuration item `maxwait` from its default of 0.5 to 0.1 and I have not had a VATSIM disconnect since.  I don't know if another value between 0.1 and 0.5 would also work.

This value determines how long to wait for an image.  Longer values will cause stuttering (and VATSIM disconnects), but results in higher-quality tiles.  Smaller values will avoid stuttering, but will result in lower-quality tiles.  I suspect that the lower-quality tiles would be the ones out on the horizon or rather far away so perhaps the lower quality would not be noticeable.  I also suspect that these lower quality tiles would be replaced by better ones once you get closer to that tile as XP requests a larger miplevel.  So the quality issue isn't all that bad.  Besides, if you are in VATSIM, this sort of quality issue may not be of concern.

I'm guessing a lot.  But this is what I gathered from reading a lot of posts and looking a little at the code.

Anyway, hope this helps.  It definitely made a difference for me.  And I can't say for sure it would help others and I think that all this depends a bit on the relative speeds of your CPU, network, disk I/O, etc.

**Reply by PilotFidi** [reply]
Hey, thanks for the reply! Ill try it out right away, can you help me where to find maxwait, and the AO configuration? m trying to find it but wasnt able to so far...?

**Reply by karlrado** [reply]
There should be an AO configuration "dotfile" called `.autoortho` in your Windows home directory.  You may have to change your file explorer settings to see it there.  Else, look for it via a terminal shell.  Open it in an editor and look for "maxwait".  There should be some comments there that further explain what it does.

With all that being said, I set mine back to the default and didn't get a VATSIM timeout.  I was flying in the same area though, and perhaps the tiles were already in the cache.  And I did see the frame rate drop near the place where it used to disconnect, but not enough for VATSIM to notice.  I mention this only to suggest that this is less likely to solve your problem since I couldn't get a positive reproduction of the issue.  Still, this is easy for you to give it a shot to see if it helps.
